index.html

```diff
@@ -7,116 +7,107 @@
+      <section class="advantages-section">
+        <h2 class="visually-hidden">Рекомендации для туристов</h2>
+          <ul class="advantages-list">
+            <li class="advantages-item">
+              <h4 class="advantages-title">Жильё</h4>
+              <p class="advantages-description">Рекомендуем пожить в настоящем мотеле, всё как в кино!</p>
             </li>
-            <li class="recommendations-item">
-              <h4 class="recommendations-title">Еда</h4>
-              <p class="recommendations-description">Всегда заказывайте топовый фирменный бургер, вы не разочаруетесь!</p>
+            <li class="advantages-item">
+              <h4 class="advantages-title">Еда</h4>
+              <p class="advantages-description">Всегда заказывайте топовый фирменный бургер, вы не разочаруетесь!</p>
             </li>
-            <li class="recommendations-item">
-              <h4 class="recommendations-title">Сувениры</h4>
-              <p class="recommendations-description">Не только китайского, но и настоящего местного производства!</p>
+            <li class="advantages-item">
+              <h4 class="advantages-title">Сувениры</h4>
+              <p class="advantages-description">Не только китайского, но и настоящего местного производства!</p>
             </li>
           </ul>
       </section>
-      <section class="booking">
-        <h2 class="booking-header visually-hidden">Бронирование гостиницы в Седоне</h2>
-        <h3 class="booking">Заинтересовались?</h3>
-        <p class="booking">Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в Седоне</p>
-          <form class="booking-button" method="post">
-              <input class="booking-button" type="submit" name="booking" value="Поиск гостиницы в Седоне">
-          </form>
+      <section class="booking-section">
+        <h2 class="visually-hidden">Бронирование гостиницы в Седоне</h2>
+        <p class="booking-caption">Заинтересовались?</p>
+        <p class="booking-subtitle">Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+        <a class="button-hotel" href="#">Поиск гостиницы в Седоне</a>
       </section>
-      <section class="mailing">
+      <section class="mailing-section">
          <h2 class="mailing-header">Подпишитесь на рассылку</h2>
          <p class="mailing-description">Только полезная информация и никакого спама, честное бойскаутское!</p>
-         <form class="mailing-button" method="get">
-              <input class="mailing-button" type="text" name="mail" value="Ваш e-mail">
-              <input class="mailing-button" type="submit" name="subscription" value="Подписаться">
+         <form class="mailing-button" method="post" action href="https://echo.htmlacademy.ru/">
+          <label> Ваш e-mail
+              <input class="field-area" type="text" name="email" placeholder="Ваш e-mail">
```

Comment:
Есть специальный тип для почты
```suggestion
              <input class="field-area" type="email" name="email" placeholder="Ваш e-mail">
```